src/js/components/plans/ctaButton.tsx

```diff
@@ -311,12 +317,13 @@ class CtaButton extends React.Component<
       selectedSteps,
     } = this.props;
     const { preflightModalOpen } = this.state;
+    const usesBothOrgTypes = plan.supported_orgs === SUPPORTED_ORGS.Both;
 
     if (user && !user.org_type) {
       return (
         <LoginBtn
           id="org-not-allowed-login"
-          label={i18n.t('Log in with a different org')}
+          label={i18n.t('Log In With a Different Org')}
```

Comment:
Do we already have translations for the new capitalization? If not, please only change the default translation, not the name of the label.

Reply:
We already had a translation for the capitalized version, so this should not break any existing translations.